device-types/Dell/PowerSwitch-N3048.yaml

```diff
@@ -5,13 +5,23 @@ slug: dell-powerswitch-n3048
 part_number: N3048
 u_height: 1
 is_full_depth: true
-comments: '[Dell EMC PowerSwitch N3000 Series 1GbE Layer 3 Switches](https://www.dell.com/en-us/shop/productdetailstxn/networking-n3000-series)'
+weight: 13.89
+weight_unit: lb
+# description: 48-port 1Gbase-T Ethernet Switch with 2 10Gbps SFP+ and 2 stacking ports, OS6
+comments: |
+  Documentation
+
+  - [Product Support page](https://www.dell.com/support/home/en-us/product-support/product/networking-n3000-series/overview)
 console-ports:
   - name: Console
     type: rj-45
 power-ports:
-  - name: PS1
+  - name: PS-1
     type: iec-60320-c14
+    maximum_draw: 75
+  - name: PS-2
+    type: iec-60320-c14
+    maximum_draw: 75
```

Comment:
Please correct me if I am wrong, but it looks like the PSUs are modular. If that is true, these should be defined as `module-bays` and the associated PSUs should be created in the module-types folder